example/spec/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -338,7 +337,8 @@ describe('When there are granule differences and granule reconciliation is run',
         }
       }).promise(),
       deleteFolder(config.bucket, testDataFolder),
-      cleanupCollections(config.stackName, config.bucket, collectionsDir),
+      // leave collections in the table for EMS reports
+      //cleanupCollections(config.stackName, config.bucket, collectionsDir),
```

Comment:
We can't rely on the order of tests, so if there's info we need in a future test we have to put it in the setup.

Reply:
Or is this for the nightly report to generate something?

Reply:
This is for nightly EMS report to be generated.  This MYD13Q1.006 collection is now only used for reconciliation report and ems report, and collection name is not modified.

Reply:
I have used a new collection MOD14A1.006 for EMS report, and put the cleanupCollections back to CreateReconciliationReportSpec.js.